index.html

```diff
@@ -268,6 +268,77 @@ <h4 class="experience-title">Skills</h4>
         </li>
       </ul>
     </section>
+    <section class="contact-form">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+      <form class="form" action="https://formspree.io/f/xknkyppy" method="POST">
+        <input
+          type="text"
+          name="name"
+          maxlength="30"
+          placeholder="Full name"
+          required
+        />
+        <input type="email" name="email" placeholder="Email address" required />
+        <textarea
+          maxlength="500"
+          name="message"
+          placeholder="Enter your text here"
+          required
+        ></textarea>
+        <button type="submit">Get in touch</button>
+      </form>
```

Comment:
- [ ] As requested by the former code reviewer, and as part of the project requirements, please make sure the "Get in touch" button styling: Active, Hover and Pressed (see design) is as specified in he design. 
- [OPTIONAL] Please the form has no styling, so is the button, it seems your styles are not properly linked, please kindly fix that. 
***The design***

![wed2](https://user-images.githubusercontent.com/65852621/134264857-4edd896c-8cf8-4e8f-aff5-2e8e0ac7ae90.png)

***Your implementation***
![yourswed1](https://user-images.githubusercontent.com/65852621/134264876-993a4788-5094-42fb-91ee-e918e210afdf.png)

Reply:
done